src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -751,7 +796,18 @@ private void ParseRuleInfo()
                             string groupName = _reader.GetAttribute(_strings.NameAttribute);
                             switch (groupName)
                             {
+                                case "Accuracy":
+                                case "Impact":
+                                case "Probability":
+                                {
+                                    _reader.Read();
+                                    string nodeValue = _reader.Value;
+                                    rule.SetProperty<string>(groupName, nodeValue);
```

Comment:
><string> [](start = 52, length = 8)

You don't have to specify the generic type parameter name. The compiler can deduce it from the type of the argument `nodeValue`, a string.